server/app/views/applicant/ApplicantCommonIntakeUpsellTemplate.html

```diff
@@ -0,0 +1,57 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+    <main role="main">
+      <h1
+        th:text="#{title.commonIntakeConfirmationTi}"
+        th:if="${isTrustedIntermediary}"
+      ></h1>
+      <h1
+        th:text="#{title.commonIntakeConfirmation}"
+        th:if="${!isTrustedIntermediary}"
+      ></h1>
+
+      <div th:if="${eligiblePrograms.size()==0}" style="margin-bottom: 20px">
+        <p
+          th:utext="#{content.commonIntakeNoMatchingProgramsTi(${moreResourcesLinkHtml})}"
+          th:if="${isTrustedIntermediary}"
+        ></p>
+        <p
+          th:utext="#{content.commonIntakeNoMatchingPrograms(${moreResourcesLinkHtml})}"
+          th:if="${!isTrustedIntermediary}"
+        ></p>
+      </div>
+
+      <div th:if="${eligiblePrograms.size()>0}" style="margin-bottom: 20px">
+        <p
+          th:text="#{content.commonIntakeConfirmationTi}"
+          th:if="${isTrustedIntermediary}"
+        ></p>
+        <p
+          th:text="#{content.commonIntakeConfirmation}"
+          th:if="${!isTrustedIntermediary}"
+        ></p>
+
+        <div th:each="ep : ${eligiblePrograms}">
+          <h2 th:text="${ep.getTitle()}">Program Name</h2>
```

Comment:
Remove "Program Name" and "Program Description" since they are getting replaced by thymeleaf anyway